packages/ra-ui-materialui/src/auth/AuthError.tsx

```diff
@@ -19,7 +20,7 @@ export const AuthError = (props: AuthErrorProps) => {
             <div className={AuthErrorClasses.message}>
                 <h1>{translate(title, { _: title })}</h1>
                 <div>{translate(message, { _: message })}</div>
-                <Button to="/login" label="ra.auth.sign_in">
+                <Button component={Link} to="/login" label="ra.auth.sign_in">
```

Comment:
I believe this change is no longer necessary?

Reply:
This writing is the new behaviour so it seems to be good to use it.

Reply:
I believe this didn't work before this PR anyway as the `Button` did not set the `component` prop automatically